concepticondata/conceptlists/Loukotka-1954-45.tsv

```diff
@@ -0,0 +1,46 @@
+ID	NUMBER	ENGLISH	SPANISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Loukotka-1954-45-1	1	head	cabeza	1256	HEAD
+Loukotka-1954-45-2	2	tongue	lengua	1205	TONGUE
+Loukotka-1954-45-3	3	tooth	diente	1380	TOOTH
+Loukotka-1954-45-4	4	eye	ojo	1248	EYE
+Loukotka-1954-45-5	5	ear	oreja	1247	EAR
+Loukotka-1954-45-6	6	hand	mano	1277	HAND
+Loukotka-1954-45-7	7	foot	pie	1301	FOOT
+Loukotka-1954-45-8	8	water	agua	948	WATER
+Loukotka-1954-45-9	9	fire	fuego	221	FIRE
+Loukotka-1954-45-10	10	sun	sol	1343	SUN
+Loukotka-1954-45-11	11	moon	luna	1313	MOON
+Loukotka-1954-45-12	12	star	estrella	1430	STAR
+Loukotka-1954-45-13	13	earth	tierra	1228	EARTH (SOIL)
+Loukotka-1954-45-14	14	stone	piedra	857	STONE
+Loukotka-1954-45-15	15	house	casa	1252	HOUSE
+Loukotka-1954-45-16	16	pot	olla	1462	POT
+Loukotka-1954-45-17	17	bow, or blowgun	areo, o bodoquera		
+Loukotka-1954-45-18	18	arrow	flecha	977	ARROW
+Loukotka-1954-45-19	19	ax	hacha	677	AXE
+Loukotka-1954-45-20	20	canoe	canoa	1970	CANOE
+Loukotka-1954-45-21	21	man	hombre	1554	MAN
+Loukotka-1954-45-22	22	woman	mujer	962	WOMAN
+Loukotka-1954-45-23	23	dog	perro	2009	DOG
+Loukotka-1954-45-24	24	wildcat	tigre		
```

Comment:
Hehe, this is a classic. You remember Yaguar in Huber et al? I think I'd really be bold here and link to Yaguar, for consistency, as this is what was probably later then used by other lists. It is about a big cat thing in the woods.

Reply:
But one may add a note here, and also mention Huber, where the same thing occurs.

Reply:
That's what I thought too. I'll leave a note then in the description.